### PR TITLE
nerfs cakehat for realsies

### PR DESCRIPTION
**Pull request by nemvar**
## GAMERS

For too long have we taken the omnipotence of the cakehat for granted. We just went there... and took it, not thinking about the consequences to our actions! I say, NO MORE! The cakehat will no longer be available to the lazy and the unrobust spessman. Only truly skilled and gifted people will now get to enjoy the cakehats wonderous embrace.

## Changelog
:cl:
add: You now get the cakehat by cooking a whole birthday cake.
del: The cakehat has been removed from all maps.
tweak: Birthday cake recipe no longer requires a cakehat. It now needs a candle, sugar and caramel.
add: Metastation bar now has a new plant. Her name is Potty.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
>TWO (or more) cakehats
yikes....

**Comment by nemvar** [comment]
You're right. I should make a global list of cakehats and divide their power through the amount of cakehats. It's the only way.

**Comment by Tlaltecuhtli** [comment]
cakehat should only happen if its your birthday

**Comment by kriskog** [comment]
Rise up, Cak lovers!

**Comment by PKPenguin321** [comment]
ok but what's the actual reason for this PR since you kind of skipped over that in your OP

**Comment by nemvar** [comment]
I don't like how strong items are available roundstart. You should work to get them. I kinda said above only in a really retarded way.

**Comment by anconfuzedrock** [comment]
I like the idea of potty

**Comment by necromanceranne** [comment]
Actually I kinda hated that I needed the cake hat to make a birthday cake so frankly I'm not complaining from that perspective.

**Comment by tralezab** [comment]
it's an interesting nerf sure

**Comment by wesoda25** [comment]
I liked that it required a cake hat bc it meant I could deny cunts like imsxz their favorite traitor weapon that they literally only get as a traitor as well as eating a black crayon and wearing a laser tag helmet and wearing a gas mask (@imsxz fuck you are your shitty transparent traitor routine). 

Limit cake hat to a config option for only bob oakland bc it was his outfit before all the other little bitch monkeys caught on it was good.

**Comment by Dawson1917** [comment]
i ded remove

**Comment by Dawson1917** [comment]
> 
> 
> I don't like how strong items are available roundstart. You should work to get them. I kinda said that in the above only in a really retarded way.

Cake hat does the same damage as a welder (15 burn). It's 100x easier to have a welder than be the first person to get the singular cakehat sitting in the bar each round. Stop essentially removing comedic items that aren't even that _good_ just because one lucky assistant each round will get a ghetto weapon that does less damage than a toolbox and hardly more than a fire extinguisher. There's no value in making the cakehat only come from eating a cake because cakehat will lose its le funny status the moment it stops being roundstart and will become an old reference to ss13 boomers

**Comment by Akrilla** [comment]
literal ided

**Comment by fludd12** [comment]
Look at them, they come to this kitchen when they know they are not pure. Greytide use the cake hat, but they are mere trespassers. Only I, chef, know the true power of the cake. I was cut in half, destroyed, but through it's burning light, the cake hat called to me. It brought me here and here I was reborn. We cannot blame these creatures, they are being led by a false prophet, an impostor who knows not the secrets of baking. Behold the greytide, come to downvote and desecrate this sacred PR. My brothers, did I not tell of this day? Did I not prophesize this moment? Now, I will stop them. Now I am changed, reborn through the flame of the cake hat. Forever bound to the kitchen. Let it be known, if the tiders want true salvation, they will lay down their arms, and wait for the baptism of my 15 burn. It is time. I will teach these trespassers the redemptive power of my cake hat. They will learn it's simple truth. The crew is lost, and they will resist. But I, chef, will cleanse this place of their impurity.

**Comment by Fikou** [comment]
> Cake hat does the same damage as a welder (15 burn). It's 100x easier to have a welder than be the first person to get the singular cakehat sitting in the bar each round. Stop essentially removing comedic items that aren't even that _good_ just because one lucky assistant each round will get a ghetto weapon that does less damage than a toolbox and hardly more than a fire extinguisher.

toolbox does 12 damage, fire extinguisher does 10 damage, cakehat is welder damage (which is the best damage available roundstart not counting null rod (1 in the round), wielded spear(sec will valid you for one) and fire axe(2 in the round)) but it doesnt need any fuel and its not a tiny/small item unlike the welder so it can dust limbs and heads

**Comment by ATH1909** [comment]
> > Cake hat does the same damage as a welder (15 burn). It's 100x easier to have a welder than be the first person to get the singular cakehat sitting in the bar each round. Stop essentially removing comedic items that aren't even that _good_ just because one lucky assistant each round will get a ghetto weapon that does less damage than a toolbox and hardly more than a fire extinguisher.
> 
> toolbox does 12 damage, fire extinguisher does 10 damage, cakehat is welder damage (which is the best damage available roundstart not counting null rod (1 in the round), wielded spear(sec will valid you for one) and fire axe(2 in the round)) but it doesnt need any fuel and its not a tiny/small item unlike the welder so it can dust limbs and heads

I'm pretty sure that at this point, wearing a cakehat is about as suspicious as carrying a spear around.

**Comment by Fikou** [comment]
> I'm pretty sure that at this point, wearing a cakehat is about as suspicious as carrying a spear around.

it has no inhand sprite and fits in backpacks

**Comment by PKPenguin321** [comment]
tgstation has temporarily blocked optimumtact 5 minutes ago

**Comment by wesoda25** [comment]
Oob

**Comment by imsxz** [comment]
@wesoda25 I do the black crayon and laser tag hat thing plenty to throw people off when I’m not an antag, in fact I’m almost certain that I’ve done at least the crayon in every single round I’ve played today.

I don’t go for cakehat because it’s strong I go for cakehat because it’s a funny item and it makes people angry. I usually ditch it fairly early because so much stuff outclasses it.

That said, I have nothing against this change and indeed welcome it. I just needed to defend my very fragile masculinity. Obligatory contrarian statement about how this is a buff because it allows mass producing of the overpowered evil cakehat.

just in case you didn’t read that last paragraph or have poor reading comprehension I am in favor of this PR cake man bad

File: code/modules/food_and_drinks/food/snacks_cake.dm [hunk 1]
```diff
@@ -175,10 +175,11 @@
 
 /obj/item/reagent_containers/food/snacks/store/cake/birthday
 	name = "birthday cake"
-	desc = "Happy Birthday little clown..."
+	desc = "Happy Birthday! You get an awesome gift if you eat this whole!"
 	icon_state = "birthdaycake"
 	slice_path = /obj/item/reagent_containers/food/snacks/cakeslice/birthday
 	slices_num = 5
+	trash = /obj/item/clothing/head/hardhat/cakehat //super sekrit club
```
Comment (81Denton) [inline code/modules/food_and_drinks/food/snacks_cake.dm]: Replace this with crafting/cooking, it makes no sense that you eat a whole cake and then a cake hat appears out of nowhere.

Comment (nemvar) [reply code/modules/food_and_drinks/food/snacks_cake.dm]: A little nonsense has never hurt this codebase, alright fine, it's now on microwave act.

Comment (kriskog) [reply code/modules/food_and_drinks/food/snacks_cake.dm]: Forcing you to eat the cake gates this behind getting fat/having to forcefeed the cake to someone though. While nonsensical, it makes it harder to do than just plain crafting.

**Pull request merged**